packages/tables/resources/views/components/header/description.blade.php

```diff
@@ -1,3 +1,6 @@
-<p {{ $attributes->class(['text-gray-900 filament-tables-header-description']) }}>
+<p {{ $attributes->class([
```

Comment:
Actually we could just fall back to the default text color like the table heading does.

```suggestion
<p {{ $attributes->class(['filament-tables-header-description']) }}>
```